build-system/global-configs/experiments-config.json

```diff
@@ -3,7 +3,7 @@
     "name": "ANALYTICS_VENDOR_SPLIT",
     "command": "gulp generate-vendor-jsons && gulp dist --defineExperimentConstant=ANALYTICS_VENDOR_SPLIT",
     "issue": "",
-    "expirationDateUTC": "2019-08-08",
+    "expirationDateUTC": "2020-08-08",
```

Comment:
: ) Thank you Esther! While this is not documented yet because we haven't official announced the support. But the idea is that each experiment should be limited to a month. Can we change it to 2019-09-08 instead? Thanks

Reply:
Sure - but when this expires it will fail our builds because `gulp update-packages` fails. I'm guessing that's not desired behavior?

Reply:
Interesting. It should fail `gulp build`, but I don't understand why it would fail `gulp update-packages`. 
We are expecting to run the experiment for 2 weeks. One month should be enough. I think the idea is to update the experiment data with another PR if we are going to run the experiment longer.

Reply:
Agreed, an expired experiment shouldn't fail unrelated `gulp` tasks. The check could be moved to its own task that's run separately on Travis.

Reply:
Agreed to only throw error when there's `--defineExperimentConstant` flag exist.